mobile_config/src/gateway_service.rs

```diff
@@ -191,27 +198,39 @@ impl mobile_config::Gateway for GatewayService {
         self.verify_request_signature(&signer, &request)?;
 
         let pool = self.metadata_pool.clone();
+        let mc_pool = self.mobile_config_db_pool.clone();
```

Comment:
the original `pool` variable name made sense here as-is when it was the _only_ database connection this function needed but in general i'd lean more towards the app's "owned" database connection more generically and the one "owned" by an outside service (the metadata db in this case) with more specificity the way we do in the config service's settings or else just give them _both_ more specific names to avoid any confusion